README.dev.md

```diff
@@ -1,5 +1,5 @@
-# Azure Resource Manager Tools
-A JSON language extension to support Azure Resource Manager deployment templates and template language expressions.
+# Azure Resource Manager ("ARM") Tools
```

Comment:
Yeah I don't think you want me to review this 😂 I can't find any other documentation that does the `("ARM")` thing and it looks a little ugly to me...

None of these have it:
https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview
https://azure.microsoft.com/en-us/features/resource-manager/
https://docs.microsoft.com/en-us/azure/azure-resource-manager/